packages/jaeger-ui/src/site-prefix.tsx

```diff
@@ -20,7 +20,7 @@ if (!baseNode && process.env.NODE_ENV !== 'test') {
   throw new Error('<base> element not found');
 }
 
-const sitePrefix = baseNode ? baseNode.href : `${global.location.origin}/`;
+const sitePrefix = baseNode ? baseNode.href : `${((global as unknown) as TestGlobal).location.origin}/`;
```

Comment:
After the packages update the global custom declaration clashed with NodeJs types.

Reply:
NodeJS type clashes are unfortunately the result of some changes to the yarn.lock that happen during adding packages:
https://github.com/jaegertracing/jaeger-ui/pull/522/files#diff-8ee2343978836a779dc9f8d6b794c3b2R1614
remove:
```
dependencies
  "@types/node" "*"
```
and `@types/node` itself from the `yarn.lock`
then `rm -rf node_modules && yarn install` _should_ fix these type issues.